Create definition for ADHD

jvbreen1 opened the pull request:
@aguywithcode @tatianamac I rebased this with the latest and made a lot of the edits suggested in #168 . Thank you both for your feedback!

jvbreen1 committed:
Create adhd.md

jvbreen1 commented:
Oh, @lkopacz I also added a bit in the Appropriate Usage section about ADHD being the "correct" diagnosis for all subtypes, including what used to be considered ADD. Let me know if it makes sense!

bnb commented on 11ty/definitions/adhd.md:
```diff
@@ -0,0 +1,48 @@
+
+---
+
+Attention Deficit Hyperactivity Disorder (initialised to ADHD) is a neurodivergence that expresses itself as a lack of impulse control at two extremes. It is commonly identified in children by their inability to remain focused on tasks that disinterests them. At the other end of the expressions, a person with ADHD has the ability to "hyperfocus" on a task that highly interests them. The following ...
+
+- performing hyperfocused tasks for an extraordinary amount of time
+- becoming unresponsive to outside prompts (even to body signals like the need to eat or sleep)
+- repetition of tasks to "perfection" rather than "good enough"
+  
+Naturally there could be positive outcomes when someone with ADHD hyperfocuses on beneficial tasks (e.g. studying a musical piece for an upcoming recital). In contrast, hyperfocus on non-beneficial tasks (like staying up late playing video games) can have the opposite effect. ADHD is a lifelong condition that some professionals believe should not be "cured". Rather the recommendation is to help t ...
```
ADHD isn't just hyperfocus, and this section is relatively focused on that specifically.  Executive dysfunction, hyperfixation, memory issues (specifically short-term memory), atypically intense emotion, and distractibility are all pretty common elements of ADHD.

jvbreen1 replied:
That is great feedback - I will spend some time this weekend adding some more information here. Thank you!

jvbreen1 replied:
Fixing this now. Side note, it is not lost on me that this section is hyper-focused on hyperfocus 😂

tatianamac commented:
@jvbreen1 Checking in to see if you're still able to finish out this PR and address @bnb 's edits? We'd love for this term to be available so let us know how we can best support you to finish it out if you're still willing.

jvbreen1 commented:
I am! I should have time to look this weekend. Thanks for the reminder.

John

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐
On Tuesday, August 25, 2020 9:21 PM, Tatiana Mac <notifications@github.com> wrote:

> [@jvbreen1](https://github.com/jvbreen1) Checking in to see if you're still able to finish out this PR and address [@bnb](https://github.com/bnb) 's edits? We'd love for this term to be available so let us know how we can best support you to finish it out if you're still willing.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, [view it on GitHub](https://github.com/selfdefined/web-app/pull/260#issuecomment-680404696), or [unsubscribe](https://github.com/notifications/unsubscribe-auth/ABHFJ5SLNWE7VTBFBHEQMZLSCRPRDANCNFSM4PC63R4A).

jvbreen1 committed:
Add other ADHD traits

Added sections for executive dysfunction, working memory, emotional dysregulation, and distractibility.

jvbreen1 commented on 11ty/definitions/adhd.md:
```diff
@@ -0,0 +1,68 @@
+
+There are several common traits and behaviors that can be associated with ADHD:
+
+### Hyperfocus
+The following traits are examples of hyperfocus:
+
+- performing hyperfocused tasks for an extraordinary amount of time
+- becoming unresponsive to outside prompts (even to body signals like the need to eat or sleep)
+- repetition of tasks to "perfection" rather than "good enough"
+
+Naturally there could be positive outcomes when someone with ADHD hyperfocuses on beneficial tasks (e.g. studying a musical piece for an upcoming recital). In contrast, hyperfocus on non-beneficial tasks (like staying up late playing video games) can have the opposite effect. ADHD is a lifelong condition that some professionals believe should not be "cured". Rather the recommendation is to help t ...
+
+### Executive Dysfunction
+Per [ADDitude](https://www.additudemag.com/what-is-executive-function-disorder/), Executive Dysfunction affects a person's ability to "engage in goal-directed action." This typically manifests in difficulty planning, completing tasks, and staying organized. This may mean someone with ADHD frequently has trouble starting chores, for example. In other cases, people with ADHD find it difficult to br ...
+
+### Working Memory Issues
+People with ADHD often have issues with their working memory. Working memory is important for decision-making and reasoning - it allows people to keep and reference multiple thoughts quickly when synthesizing information and also when context-switching. A deficit in working memory means that people with ADHD can seem forgetful and it makes context-switching especially difficult. It can also resul ...
+
+### Emotional Dysregulation
+ADHD makes it hard to regulate emotional reactions. According to [How to ADHD](https://www.youtube.com/watch?v=2kew2JhKq3Y), Emotional Dysregulation is often overlooked, but it can be partly attributed to deficits in inhibition, self-soothing, focus, and working memory. In children, this can cause tantrums. Adults with ADHD are often expressive, excitable, and they often show atypically intense e ...
+
+### Distractibility
+Distractibility is one of the "hallmark" traits people think of when they think of ADHD. Because ADHD causes interest-driven behavior, people with ADHD often have trouble maintaining focus on uninteresting tasks. This means people with ADHD often jump between several tasks, and it can be very easy for them to become distracted by other things. Common distractors are loud sounds, other people, the ...
+
+Distractibility also has social consequences - in conversation, it can seem to others that someone with ADHD is not listening or does not care. A common behavior to counteract distractibility is ["stimming"](https://www.healthline.com/health/autism/stimming), which is commonly associated with Autism. Many people with ADHD use fidget devices or other tools to stimulate their senses and keep themse ...
```
@bnb @tatianamac I added a few extra sections here to cover some of the common behaviors and traits of ADHD. I'm having a bit of difficulty gathering my thoughts into words right now (thanks, ADHD!), so I don't know how accurate my descriptions are. I could use some help/guidance here on this section.

bnb replied:
This LGTM. Also had to laugh when I was reading the stimming part because I was fidgeting with my AirPods case while reading.

jvbreen1 replied:
😂 I had several such moments while working on this. Thanks for the follow up!

bnb reviewed (approved):
This LGTM. This is a solid piece.

jvbreen1 committed:
Merge branch 'prod' into main

jvbreen1 committed:
Fixed Markdown Lint issues

jvbreen1 commented:
@tatianamac I fixed the linting issues that were causing the build to fail here. Is there anything else you need from me on this one? Otherwise, I think it might be ready to merge! Thanks for creating a project that's meaningful and for fostering a community that is helpful and welcoming.

tatianamac reviewed (approved):
🚀

tatianamac committed:
Merge branch 'prod' into main

tatianamac committed:
Merge branch 'prod' into main

tatianamac commented:
Thank you everyone, especially @jvbreen1, for contributing to this important definition. I'm looking forward to sharing this one! Please excuse the delay in merging on my part 🙃

tatianamac committed:
Merge branch 'prod' into main

tatianamac committed:
Merge branch 'prod' into main

jvbreen1 commented:
Thanks for merging! No worries at all on the delay - this is your project and I'm just happy to be able to contribute to it.
